particle-iot/device-os
# Change the prefix of BLE broadcasting name for B5SoM.

### Problem

The prefix of BLE broadcasting name is `B5som-`, while we want it to be `Boron-`

### Solution

Refactor the `get_device_name()` API in HAL.

### Steps to Test

1. Build and flash the test application: `make PLATFORM=b5som TEST=app/ble/uart_peripheral`
2. Use mobile App to scan BLE peripherals and see if your B5SoM is broadcasting as `Boron-xxxxxx`.

### Example App

N/A

### References

N/A

---

### Completeness
